### PR TITLE
Add resolving DID Document in VDR Registry when generating presentation by id.

### PR DESCRIPTION
**Pull request by msjeong482**
**Title:**
Add resolving DID Document in VDR Registry.

**Summary:**
When generating a presentation by ID, DID Documents are searched only in local storage, so even though DID Documents are in the VDR Registry, the process of storing DID Documents in local storage should have been additionally performed.
So, I added the process of finding DID Document in VDR Registry.

### CODE REVIEW

File: pkg/controller/command/verifiable/command.go [hunk 1]
```diff
@@ -711,16 +711,24 @@ func (o *Command) GeneratePresentationByID(rw io.Writer, req io.Reader) command.
 		return command.NewValidationError(GeneratePresentationByIDErrorCode, fmt.Errorf("get vc by id : %w", err))
 	}
 
-	doc, err := o.didStore.GetDID(request.DID)
+	var didDoc *did.Doc
```
Comment (sudeshrshetty) [inline pkg/controller/command/verifiable/command.go:714]: I think it should be other way around. If someone caches DID then local storage should be looked first.

Comment (msjeong482) [reply pkg/controller/command/verifiable/command.go]: Thank you for sharing your good opinion. If the DID is cached, it has been modified to look in the local storage first.

I have an additional question.
When I look into the parts of DID search in many protocols, I look into the VDR Registry first, and if not, look into local storage. (I also referred to that part.)
Does this need to be modified?

**Review by sudeshrshetty — APPROVED** [review]

**Review by fqutishat — APPROVED** [review]

File: pkg/controller/command/verifiable/command.go (hunk 1 above)
Comment (sudeshrshetty) [reply pkg/controller/command/verifiable/command.go]: @msjeong482 looks good to me now. Please rebase so that we can merge this PR.

**Comment by fqutishat** [comment]
@msjeong482 please squash your commits to one commit

**Pull request merged**